PowerPointLabs/PowerPointLabs/TimerLab/TimerLabPaneWPF.xaml.cs

```diff
@@ -438,28 +438,20 @@ private void WidthSlider_ValueChanged(object sender, RoutedPropertyChangedEventA
                 var timerBody = GetShapeByName(TimerLabConstants.TimerBodyId);
                 float increment = value - timerBody.Width;
 
-                if (increment == 0)
-                {
-                    ReformMarkersIfMissing();
-                }
-                else
-                {
-                    var lineMarkerGroup = GetLineMarkerGroup();
-                    var timeMarkerGroup = GetShapeByName(TimerLabConstants.TimerTimeMarkerGroupId);
-                    timerBody.Select();
-                    if (lineMarkerGroup != null)
-                    {
-                        lineMarkerGroup.Select(Microsoft.Office.Core.MsoTriState.msoFalse);
-                    }
-                    timeMarkerGroup.Select(Microsoft.Office.Core.MsoTriState.msoFalse);
+                timerBody.Left = NewPosition(timerBody.Left, increment);
+                timerBody.Width = timerBody.Width + increment;
 
-                    var tempGroup = this.GetCurrentSelection().ShapeRange.Group();
```

Comment:
The components were grouped together so that it is less choppy it is resizing. Does it become more choppy when you move these 3 components separately?